server/design/toolmetrics/design.go

```diff
@@ -54,6 +54,44 @@ var _ = Service("logs", func() {
 		Meta("openapi:extension:x-speakeasy-react-hook", `{"name": "ListToolLogs"}`)
 	})
 
+	Method("listToolExecutionLogs", func() {
+		Description("List structured logs from tool executions.")
+		Security(security.ByKey, security.ProjectSlug)
+		Security(security.Session, security.ProjectSlug)
+
+		Payload(func() {
+			Extend(ListToolExecutionLogsPayload)
+			security.ByKeyPayload()
+			security.SessionPayload()
+			security.ProjectPayload()
+		})
+
+		Result(ListToolExecutionLogsResult)
+
+		HTTP(func() {
+			GET("/rpc/logs.listToolExecutionLogs")
+			security.ByKeyHeader()
+			security.SessionHeader()
+			security.ProjectHeader()
+
+			Param("ts_start")
+			Param("ts_end")
+			Param("deployment_id")
+			Param("function_id")
+			Param("instance")
+			Param("level")
+			Param("source")
+			Param("cursor")
+			Param("per_page")
+			Param("direction")
+			Param("sort")
+		})
+
+		Meta("openapi:operationId", "listToolExecutionLogs")
+		Meta("openapi:extension:x-speakeasy-name-override", "listToolExecutionLogs")
+		Meta("openapi:extension:x-speakeasy-react-hook", `{"name": "ListToolExecutionLogs"}`)
```

Comment:
```suggestion
		Meta("openapi:extension:x-speakeasy-react-hook", `{"name": "ToolExecutionLogs"}`)
```

will map to `useToolExecutionLogs` instead of `useListToolExecutionLogs`